src/workerd/jsg/setup.c++

```diff
@@ -252,11 +252,14 @@ HeapTracer& HeapTracer::getTracer(v8::Isolate* isolate) {
   return IsolateBase::from(isolate).heapTracer;
 }
 
+// todo: cleanup after 13.2 upgrade
+#if V8_MINOR_VERSION < 2
```

Comment:
Mind adding a check for V8_MAJOR_VERSION == 13 as well please?

Reply:
We can add #error if v8 isn't 13 or minor_version > 2 just to make sure we can't forget to clean this up

Reply:
sure, done. Though I wanted it to rot if we don't clean it up soon enough.